python/orca/src/bigdl/orca/learn/tf2/ray_estimator.py

```diff
@@ -302,15 +301,15 @@ def fit(self,
                                                                                  data_config)
                     remote_worker_stats.append(self.remote_workers[i].step.remote(**params))
                 worker_stats = ray.get(remote_worker_stats)
-                worker_stats = list(itertools.chain.from_iterable(worker_stats))
         else:
             params["data_creator"] = data
             params["validation_data_creator"] = validation_data
             params_list = [params] * self.num_workers
 
             worker_stats = ray.get([self.remote_workers[i].step.remote(**params_list[i])
                                     for i in range(self.num_workers)])
-            worker_stats = list(itertools.chain.from_iterable(worker_stats))
+        # Tensorflow automatically synchronizes results on all the wokers
+        # thus only need to save the first one
```

Comment:
Tensorflow -> TensorFlow
save the first return -> return the result of the first worker